webhookValidationConfig.go

```diff
@@ -53,63 +37,20 @@ type TTLVConfig struct {
 	Now    func() time.Time
 }
 
-// BuildValidURLFuncs translates the configuration into a list of ValidURLFuncs
-// to be run on the webhook.
-func buildValidURLFuncs(config ValidatorConfig) ([]ValidURLFunc, error) {
-	var v []ValidURLFunc
-	v = append(v, GoodURLScheme(config.URL.HTTPSOnly))
-	if !config.URL.AllowLoopback {
-		v = append(v, RejectLoopback())
-	}
-	if !config.URL.AllowIP {
-		v = append(v, RejectAllIPs())
-	}
-	if !config.URL.AllowSpecialUseHosts {
-		config.URL.InvalidHosts = append(config.URL.InvalidHosts, SpecialUseHosts...)
-	}
-	if len(config.URL.InvalidHosts) > 0 {
-		v = append(v, RejectHosts(config.URL.InvalidHosts))
-	}
-	if !config.URL.AllowSpecialUseIPs {
-		config.URL.InvalidSubnets = append(config.URL.InvalidSubnets, SpecialUseIPs...)
-	}
-	if len(config.URL.InvalidSubnets) > 0 {
-		fInvalidSubnets, err := InvalidSubnets(config.URL.InvalidSubnets)
-		if err != nil {
-			return nil, fmt.Errorf("%w: %v", errFailedToBuildValidURLFuncs, err)
-		}
-		v = append(v, fInvalidSubnets)
-	}
-	return v, nil
-}
-
-// BuildValidators translates the configuration into a list of validators to be run on the
-// webhook.
-func BuildValidators(config ValidatorConfig) (Validators, error) {
-	v, err := buildValidURLFuncs(config)
-	if err != nil {
-		return nil, fmt.Errorf("%w: %v", errFailedToBuildValidators, err)
+// BuildURLChecker translates the configuration into url Checker to be run on the webhook.
+func (config *ValidatorConfig) BuildURLChecker() (*urlegit.Checker, error) {
```

Comment:
I'm not seeing where `config.IP.ForbiddenSubnets`, `config.Domain.ForbiddenDomains`, `config.TTL` are being used... where are they suppose to be used?

Reply:
that's my bad - forgot to add in for first two. config.TTL is used when building the validators which will be in the next PR.

Reply:
ok sounds good